chainer/training/extensions/snapshot_writers.py

```diff
@@ -57,6 +57,29 @@ def save(self, filename, outdir, target, savefun, **kwds):
             savefun(tmppath, target)
             shutil.move(tmppath, os.path.join(outdir, filename))
 
+        self._post_save(filename, outdir, target, savefun, **kwds)
+
+    def _add_cleanup_hook(self, hook_fun):
+        """Adds post-save hook function.
```

Comment:
Please update the comment. It is no longer exposed to users and it looks some sentences are obsolete.